CalendarApp/NUnitTestProject/BaseIntegrationTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using NUnit.Framework;
+using NUnitTestProject.Common;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+using Google.Apis.Calendar.v3.Data;
+using BookingRoom.Models.GoogleCalendar;
+using BookingRoom.Models.GoogleEvent;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public class BaseIntegrationTest
+    {
+        protected IServiceLocator _serviceLocator;
+        protected IUnityContainer _unityContainer;
+        protected BookingService _meetingBooking;
+
+        [TestFixtureSetUp]
+        public void Initialize()
+        {           
+            _unityContainer = Dependency.UnityConfig.GetUnityContainer();
+            _serviceLocator = _unityContainer.Resolve<IServiceLocator>();
```

Comment:
Just curious: what is the reason of recursive dependency between `IServiceLocator` and `Unity DI` in general?
Another possible solution is just wrap `Unity DI` by `IServiceLocation`-implementation or simple `Unity DI`-usage.